frigate/video.py

```diff
@@ -156,6 +164,32 @@ def get_enabled_state():
 
             frame_rate.update()
 
+            # Update stall metrics
```

Comment:
I don't think we want to do this within the frame management logic of the camera itself, we don't want to do anything that can add any delay here, and we can have stalls for other reasons than the camera itself.

I think this would be better to do either:
- in the watchdog, we can use the existing checks and just have a smaller check which looks for the last processed frame value to mark something as stalled.
- in a separate thread which just subscribes to the camera frames, similar to what the object detector already does

Reply:
sounds good, will tweak